systemd/opentofu-runner.service

```diff
@@ -1,16 +1,25 @@
 [Unit]
+Description=Opentofu Runner Container
+RequiresMountsFor=%t/containers
+RequiresMountsFor=/var/lib/manageiq/opentofu-runner/certs
 PartOf=opentofu-runner.target
+
 [Install]
 WantedBy=opentofu-runner.target
+
 [Service]
+Restart=on-failure
 User=manageiq
 Group=manageiq
-ExecStartPre=/bin/rm -f /tmp/%n.cid
-ExecStart=/usr/bin/podman run --conmon-pidfile %T/%N.pid --cidfile %T/%N.cid --cgroup-manager=cgroupfs --cgroups=no-conmon --log-driver=journald --replace --name=opentofu-runner --secret=opentofu-runner-secret --root=/var/www/miq/vmdb/data/containers/storage --env=DATABASE_HOSTNAME=${DATABASE_HOSTNAME} --env=DATABASE_NAME=${DATABASE_NAME} --env=DATABASE_USERNAME=${DATABASE_USERNAME} --env=MEMCACHE_SERVERS=${MEMCACHE_SERVERS} --volume=/var/lib/manageiq/opentofu-runner/certs:/opt/app-root/src/config/cert:z --expose=6000 --net=host $OPENTOFU_RUNNER_IMAGE
-ExecStop=/usr/bin/podman stop --ignore -t 30 --cidfile %T/%N.cid --cgroup-manager=cgroupfs --root=/var/www/miq/vmdb/data/containers/storage
-ExecStopPost=/usr/bin/podman rm --ignore --cidfile %T/%N.cid --cgroup-manager=cgroupfs --root=/var/www/miq/vmdb/data/containers/storage
-ExecStopPost=/usr/bin/rm -f %T/%N.pid %T/%N.cid
-KillMode=none
-Type=simple
-Restart=always
 Slice=manageiq.slice
+Environment=PODMAN_SYSTEMD_UNIT=%n
+KillMode=mixed
+Delegate=yes
+Type=notify
+NotifyAccess=all
+SyslogIdentifier=%N
+ExecStartPre=/bin/rm -f %T/%N.cid
+ExecStartPre=loginctl enable-linger manageiq
+ExecStart=/usr/bin/podman run --name=opentofu-runner --cidfile=%T/%N.cid --replace --rm --cgroups=split --network=host --sdnotify=conmon -d -v /var/lib/manageiq/opentofu-runner/certs:/opt/app-root/src/config/cert:z --env DATABASE_HOSTNAME=${DATABASE_HOSTNAME} --env DATABASE_NAME=${DATABASE_NAME} --env DATABASE_USERNAME=${DATABASE_USERNAME} --env MEMCACHE_SERVERS=${MEMCACHE_SERVERS} --env PORT=${PORT} --expose ${PORT} --secret opentofu-runner-secret --root /var/www/miq/vmdb/data/containers/storage ${OPENTOFU_RUNNER_IMAGE}
+ExecStop=/usr/bin/podman kill --root /var/www/miq/vmdb/data/containers/storage --cidfile=%T/%N.cid
```

Comment:
NOTE I switched to podman kill here because the opentofu-runner image doesn't stop cleanly causing timeout and a SIGKILL anyway, we should fix this and is being tracked by https://github.com/ManageIQ/manageiq-providers-embedded_terraform/issues/38